docs/Customization.md

```diff
@@ -0,0 +1,21 @@
+# UI Customization
+
+## iOS
+
+Create swift class 'AdyenAppearance' extending protocol 'AdyenAppearanceProvider'.
+SDK will use reflection to find the class with this exact name.
+
+```swift
+import Adyen
+import adyen_react_native
+
+class AdyenAppearance: AdyenAppearanceProvider {
+  static func createStyle() -> Adyen.DropInComponent.Style {
+     # provide your custom style here
+  }
+}
+```
+
+## Android
+
+Locate correspondent resources from [Android SDK](https://github.com/Adyen/adyen-android/tree/4.10.0) and override them in the local `android\app\src\main\res` folder according to your needs.
```

Comment:
Our latest is `4.11.0` and we're releasing again soon, I think it's maybe better to replace `4.10.0` with `YOUR_VERSION` or something?

Reply:
This is URL in dock.. not sure it is possible to store "local variable" per branch...

Reply:
Updated on [f1aae01](https://github.com/Adyen/adyen-react-native/pull/212/commits/f1aae018044beb765fbfd888dd57ed02e10b7be2)